src/frontend-proxy.ts

```diff
@@ -62,8 +63,15 @@ export async function frontendProxy(
       ? Math.random()
       : cookieValue
 
+    let contentApiRequest = null
+
+    if (url.hasContentApiParameters()) {
+      url.pathname = '/content-only' + url.pathname
+      contentApiRequest = new Request(url.toString(), new Request(request))
```

Comment:
this way we don't have proxy and use the same logic as before. But I'm open to other approaches.